### DISCUSSION TITLE
useFilenamingConvention and file based routing catch all `[...slug]`

### DISCUSSION THREAD

**Discussion by minht11** [biomejs/biome#3444] [Ideas]
Many popular file based routers provide option to catch all routes.
[Next](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#catch-all-segments)
[Nuxt](https://nuxt.com/docs/guide/directory-structure/server#catch-all-route)
[Astro](https://docs.astro.build/en/guides/routing/#rest-parameters)
and I plenty more.

I might be wrong but disabling rule only for those files might not be possible because glob `"[...*].svelte"` would not work.

Given the popularity of the use case, I think we should support it by default, so users using lint rule need less config.

One option would be allowing `...` when they start inside brackets.

**Comment by Conaclos** [comment]
I am closing the discussion because I opened the issue #3465 and the PR #3466 to support this.
